firefox-ios/firefox-ios-tests/Tests/XCUITests/TrackingProtectionTests.swift

```diff
@@ -178,4 +178,31 @@ class TrackingProtectionTests: BaseTestCase {
         mozWaitForElementToExist(app.staticTexts.elementContainingText("Firefox has not connected to this website."))
         XCTAssertEqual(app.buttons[AccessibilityIdentifiers.Toolbar.trackingProtection].label, "Connection not secure")
     }
+
+    // https://mozilla.testrail.io/index.php?/cases/view/2693741
+    func testLockIconCloseMenu() {
+        navigator.openURL("https://www.Mozilla.org")
```

Comment:
I will leave mozilla.org website, because our local servers have the guard turned off and the test will benefit more having a real simulation of tracking protection usage.